src/IssueProcessor.ts

```diff
@@ -249,8 +250,14 @@ export class IssueProcessor {
     // find any comments since the stale label
     const comments = await this.listIssueComments(issue.number, sinceDate);
 
-    // if there are any user comments returned, issue is not stale anymore
-    return comments.filter(comment => comment.user.type === 'User').length > 0;
+    // if there are any user comments returned, and they were not by this bot, the issue is not stale anymore
+    return (
+      comments.filter(
+        comment =>
+          comment.user.type === 'User' &&
+          comment.user.login !== github.context.actor
```

Comment:
Seems like others have ran into the same issue: https://github.com/actions/stale/issues/21#issuecomment-656002085

Reply:
I've opened https://github.com/actions/stale/pull/192 where I propose we stop filtering out comments by `github.context.actor` when determining if there has been activity on the PR since it was marked as stale.